Realm/Swift/RLMSupport.swift

```diff
@@ -19,32 +19,55 @@
 import Realm
 
 extension RLMRealm {
+    /**
+     Swift wrapper for `+ [RLMRealm schemaVersionAtURL:`.
+     - see `+ [RLMRealm schemaVersionAtURL:]`
+     */
     @nonobjc public class func schemaVersion(at url: URL, usingEncryptionKey key: Data? = nil) throws -> UInt64 {
         var error: NSError?
         let version = __schemaVersion(at: url, encryptionKey: key, error: &error)
         guard version != RLMNotVersioned else { throw error! }
         return version
     }
 
+    /**
+     Swift wrapper for ` - [RLMRealm resolveThreadSafeReference:]`.
+     - see `- (nullable id)resolveThreadSafeReference:`
+     */
     @nonobjc public func resolve<Confined>(reference: RLMThreadSafeReference<Confined>) -> Confined? {
         return __resolve(reference as! RLMThreadSafeReference<RLMThreadConfined>) as! Confined?
     }
 }
 
 extension RLMObject {
-    // Swift query convenience functions
+    /**
+     Swift convenience wrapper for `+ [RLMObject objectsWithPredicate:]`.
+     - see `+ (RLMResults *)objectsWithPredicate:`
+     */
     public class func objects(where predicateFormat: String, _ args: CVarArg...) -> RLMResults<RLMObject> {
         return objects(with: NSPredicate(format: predicateFormat, arguments: getVaList(args))) as! RLMResults<RLMObject>
     }
 
+    /**
+     Swift convenience wrapper for `+ [RLMObject objectsInRealm:withPredicate:]`.
+     - see `+ (RLMResults *)objectsInRealm:withPredicate:`
+     */
     public class func objects(in realm: RLMRealm,
                               where predicateFormat: String,
                               _ args: CVarArg...) -> RLMResults<RLMObject> {
         return objects(in: realm, with: NSPredicate(format: predicateFormat, arguments: getVaList(args))) as! RLMResults<RLMObject>
     }
 }
 
-public struct RLMIterator<T>: IteratorProtocol {
+// Sequence conformance for RLMArray and RLMResults is provided by RLMCollection's
+// `makeIterator()` implementation.
+extension RLMArray: Sequence {}
+extension RLMResults: Sequence {}
```

Comment:
These extensions should be public.

Reply:
Extensions that conform to protocols can't be public:
<img width="1070" alt="Screen Shot 2020-11-06 at 1 17 35 PM" src="https://user-images.githubusercontent.com/25092858/98400920-db9ec080-2032-11eb-951b-4e98e22c4505.png">

The developer only accesses `Sequence` functionality through the `public func makeIterator()`. So something like this is still possible:
```
let rlmRealm = try! RLMRealm.default()
        try! rlmRealm.transaction {
            for i in 1...3 {
                let person = Person(value: ["age": "\(i)")
                rlmRealm.add(person)
            }
        }

        let rlmResults = Person.allObjects()
        var iterator = rlmResults.makeIterator()
        while let person = iterator.next() {
            // do something
        }
```